src/main/kotlin/rat/poison/scripts/BombTimer.kt

```diff
@@ -76,9 +78,21 @@ fun bombUpdater() = every(25, true) {
             canDefuse = false
             gettingDefused = false
         }
+        var b: Boolean? = false
+        if (curSettings["LS_BOMB"].strToBool() && me.team() == 3.toLong() && !me.hasDefuser() && bombState.timeLeftToExplode < 10.2 && b == false)
+            run {
+                CSGO.clientDLL[dwUse] = 5
+                b = true
+            } else if (curSettings["LS_BOMB"].strToBool() && me.team() == 3.toLong() && me.hasDefuser() && bombState.timeLeftToExplode < 5.1 && b == false) {
```

Comment:
What is the chance of missing this time ? Also better check if we are close to the bomb.

Reply:
I have tested it over a dozen times and I personally haven't lost it once. I can increase the time if necessary. I always get 0.0 seconds remaining or 0.03 ish. Here is a video of it: https://streamable.com/xsvpf

Reply:
I'm thinking about either increasing the frequency of that function or increasing the time to 5.6/10.6 to be on the safe side.